From 5bdda0d6b8ca0c39e03e9b4df4950d6704b35089 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20=C4=8Cerm=C3=A1k?= <dcermak@suse.com>
Date: Tue, 23 Aug 2022 16:19:11 +0200
Subject: [PATCH 10/12] Remove dependency on grpc-tools

This package is only needed to build the go plugins, which we do not use at the
moment.
---
 vagrant.gemspec | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/vagrant.gemspec b/vagrant.gemspec
index 410484dc1..c8a25cd0b 100644
--- a/vagrant.gemspec
+++ b/vagrant.gemspec
@@ -39,9 +39,6 @@ Gem::Specification.new do |s|
   s.add_dependency "winrm-elevated", ">= 1.2.1", "< 2.0"
   s.add_dependency "winrm-fs", ">= 1.3.4", "< 2.0"
 
-  # Needed for go generate to use grpc_tools_ruby_protoc
-  s.add_development_dependency "grpc-tools", "~> 1.41"
-
   # required to include https://github.com/ruby/ipaddr/issues/35
   s.add_dependency "ipaddr", ">= 1.2.4"
 
-- 
2.38.1

